Project-3/Peihong_Xie_Project_3_50375781.md

```diff
@@ -0,0 +1,381 @@
+
+**[3]** Model the following natural language expressions using terms from BFO and RO; you are welcome to introduce new terms where needed:  
+```
+  (a) Sally has an arm Tuesday but does not have an arm Wednesday. 
+  (b) Every liver has some cell as part at all times it exists.
+  (c) John was a child, then an adult, then a senior. 
+  (d) Goofus and Gallant are married at each point in a three year span. 
+```
+
+**Answer:**
+
+(a) Sally instance_of Object
+
+Arm is_a Fiat object part
+
+Tuesday/Wednesday is_a One-dimensional temporal region
+
+"having some Arm on some Tuesday" instance_of occurrent
+
+"having no Arm on some Wednesday" instance_of occurrent
+
+**Result**: "Sally participates in having some Arm on some Tuesday" precedes "Sally participates in having no Arm on some Wednesday".
+
+Note: "Sally does not have an arm" is ambiguous because it can also be read as "Sally loses an arm but has still other arms" (Maybe Sally is an octopus).
+
+(b) Liver is_a fiat object part
+
+Cell is_a material entity
+
+**Result**: Liver has_part_at_all_times Cell. 
+
+Note: **"has_part_at_all_times"** has been used by Professor Barry Smith to deal with similar cases. 
+
+(c) Childhood is_a Occurrent
+
+John's childhood instance_of Childhood
+
+Adulthood is_a Occurrent
+
+John's adulthood instance_of Adulthood
+
+Seniorhood is_a Occurrent
+
+John's seniorhood instance_of Seniorhood
+
+John instance_of Object
+
+**Result**: John participates_in John's childhood, and John participates_in John's adulthood, and John participates_in John's seniorhood, and John's childhood precedes John's adulthood, and John's adulthood precedes John's seniorhood.
+
+(d) t instance_of Zero-dimensional temporal region
+
+"This three years" instance_of One-dimensional temporal region
+
+Goofus instance_of Object
+
+Gallant instance_of Object
+
+Marriage is_a Occurrent
+
+**Result**: If any t part_of this three years, then Goofus participates_in some Marriage at t, and Gallant also participates_in some Marriage t.
+
+Note: Goofus and Gallant might not be a couple.
+
+**[4]** Using the language of First-Order Logic, represent the following natural language expressions; you are welcome to introduce new terms where needed: 
+```
+  (a) Sally has an arm Tuesday but does not have an arm Wednesday. 
+  (b) Every liver has some cell as part at all times it exists.
+  (c) John was a child, then an adult, then a senior. 
+  (d) Goofus and Gallant have been married for three years; for each day of that span, it is true to assert they are married. 
+```
+
+**Answer:**
+
+(a) Use the following key: 
+
+Has(x,y,t) : x has y at time t
+
+Arm (x) : x is an arm
+
+T(t) : t is a Tuesday 
+
+W(t) : t is a Wednesday
+
+**s** : Sally
+
+So the sentence (a) is symbolized as: **∃t∃x(Has(s,x,t) ∧ Arm(x) ∧ T(t)) ∧ ∃t∀x(W(t) ∧ (Arm (x) → ~Has(s,x,t)))**
```

Comment:
Watch out for the drinker's paradox when using existential quantifiers and material conditionals: https://en.wikipedia.org/wiki/Drinker_paradox